packages/pixeloven-react-router-config/src/components/Routes/Routes.test.tsx

```diff
@@ -23,13 +23,23 @@ const routes = [
 describe("@pixeloven/react-router-config", () => {
     describe("Components", () => {
         describe("Routes", () => {
-            it("should render one <Switch /> and two <Route /> components", () => {
+            it("should render just two <Route /> components", () => {
```

Comment:
nit: should it be just *one* Route component?

Reply:
Nope it mounts two Route components.

Reply:
This change just made this component with or with out the switch. So no matter what it should have two route components.

Reply:
And to clarify the <Route /> is not the same as the component I am testing which is <Routes />